src/ontology/mondo-ingest.Makefile

```diff
@@ -509,6 +505,38 @@ slurp-all-no-updates: $(foreach n,$(ALL_COMPONENT_IDS), slurp-no-updates-$(n))
 .PHONY: slurp-all
 slurp-all: $(foreach n,$(ALL_COMPONENT_IDS), slurp-$(n))
 
+
+#############################
+###### Synchronization ######
+#############################
+.PHONY: generate-synchronization-files
+generate-synchronization-files: sync-subclassof
+
+.PHONY: sync-subclassof
+sync-subclassof: reports/sync-subClassOf.in-mondo-only.tsv
+
+# TODO: Drop this? this isn't really needed. if so, also delete docs from here and workflows.md
+# TODO: PR comment: I dropped '-B' from this pattern in slurp-%, but otherwise re-using the pattern. idk why that contains -B. is that erroneous?
+.PHONY: sync-subclassof-%
+sync-subclassof-%:
+	$(MAKE) reports/$*.subclass.added.robot.tsv
+
+# todo: When ODK has oaklib >= 0.5.20, can remove the pip-oaklib dependency: https://github.com/INCATools/ontology-development-kit/issues/936
+reports/%.subclass.added.robot.tsv reports/%.subclass.confirmed.robot.tsv reports/%.subclass.in-mondo-only.tsv: tmp/mondo-ingest.db tmp/mondo.db tmp/mondo.sssom.tsv pip-oaklib
```

Comment:
Never include a .PHONY goal as a dependency of a make goal.
```suggestion
reports/%.subclass.added.robot.tsv reports/%.subclass.confirmed.robot.tsv reports/%.subclass.in-mondo-only.tsv: tmp/mondo-ingest.db tmp/mondo.db tmp/mondo.sssom.tsv
	$(MAKE) pip-oaklib
```

Reply:
Yes, I guess I can see the negative repercussions, even though this was temporary. I put in a request for ODK go upgrade the package, but I think it'll be a while. I think I set some kind of reminder about removing this. And I also had added this TODO above the goal:
> todo: When ODK has oaklib >= 0.5.20, can remove the pip-oaklib dependency: https://github.com/INCATools/ontology-development-kit/issues/936

I'm removing it now; however a fresh build will fail without this I think.

Reply:
> I'm removing it now; however a fresh build will fail without this I think.

Why didn't you accept my suggestion above? A fresh build should never fail!

Reply:
Ah, my mistake. I didn't see it. And of course it makes perfect sense, I should have thought of that myself. Changing it now.